src/compiler/crystal/syntax/transformer.cr

```diff
@@ -165,17 +165,7 @@ module Crystal
       node
     end
 
-    def transform(node : PointerOf)
-      node.exp = node.exp.transform(self)
-      node
-    end
-
-    def transform(node : SizeOf)
-      node.exp = node.exp.transform(self)
-      node
-    end
-
-    def transform(node : InstanceSizeOf)
+    def transform(node : UnaryExpression)
```

Comment:
**praise:** This is a great simplification! 🚀
Ideally, I'd prefer this refactor on a separate PR though because it's largely unrelated to the introduction of `alignof`.